feat: add 6 feature and 1 docs

M16Yusuf opened the pull request:
feat: feature 1 myTriangle, in tasks/yusuf/triangle.js
feat: feature 2 useCountArray, in /tasks/yusuf/countArray.js
feat: feature 3  myTriangleVer2, in /tasks/yusuf/triangleVer2.js
feat: feature 4 prosesNumbers, in /tasks/yusuf/prosesNumber.js
feat: feature 5  cetakTabel, in /tasks/yusuf/cetakTabel.js
feat: feature 6 buatRentang, in /tasks/yusuf/buatRentang.js
docs: add docs for question minitask no 7 week 2 day 3, written in index.js with comment

M16Yusuf committed:
feat: add triangle feature, minitask-week2day3no1

M16Yusuf committed:
feat: add countArray feature, minitask-week2d3no2

M16Yusuf committed:
feat: add trianlever2 feature,minitask-w2day3no3

M16Yusuf committed:
feat: add feature prosesNumbers, minitask-w2d3no4

M16Yusuf committed:
feat: add feature cetakTabel,mintask-w2d3no5

M16Yusuf committed:
feat: add feature buatRentang, minitask-w2d3no6

M16Yusuf committed:
docs: add desc/answer for minitask-w2d3No7

kodacampmain commented:
pemanggilan fungsi tugas di index.js silahkan diperbaiki
bisa melihat contoh yang sudah ada disitu
@M16Yusuf

M16Yusuf committed:
fix 3 change and 1 new feature

fix: feature cetakTabel, update error handling -> throw error to index
fix: feature triangle, update error handling -> throw error to index
fix: feature triangleVer2, update error handling -> throw error to index
feat: add new feature index.js in /task/Yusuf/ for module export import handling

M16Yusuf commented:
fix 3 change and 1 new feature
fix: feature cetakTabel, update error handling -> throw error to index
fix: feature triangle, update error handling -> throw error to index
fix: feature triangleVer2, update error handling -> throw error to index
feat: add new feature index.js in /task/Yusuf/ for module export import handling.
@kodacampmain

kodacampmain commented:
done
